src/cow-react/modules/limitOrders/state/limitOrdersAtom.ts

```diff
@@ -23,3 +40,35 @@ export function getDefaultLimitOrdersState(chainId: SupportedChainId | null): Li
 }
 
 export const limitOrdersAtom = atomWithStorage<LimitOrdersState>('limit-orders-atom', getDefaultLimitOrdersState(null))
+
+export const useLimitOrdersStateManager = (): LimitOrdersStateManager => {
+  const history = useHistory()
+  const [state, setState] = useAtom(limitOrdersAtom)
+
+  return useMemo(() => {
+    return {
+      state,
+      setState(state: LimitOrdersState) {
```

Comment:
Why exposing a single manager with all the functions instead of exporting all the actions as individual functions?